index.jsx

```diff
@@ -168,13 +168,14 @@ module.exports = function Dimensions ({
         }
         return (
           <div style={wrapperStyle} ref='wrapper'>
-            {(containerWidth || containerHeight) &&
-              <ComposedComponent
+            {(containerWidth || containerHeight)
```

Comment:
This way you are not creating the ComposedComponent if the dimensions are 0. This may lead to some confusion.
You could solve the printing of '0' by casting the above statement to a boolean like this `!!(containerWidth || containerHeight)`